react/components/Shipping/ShippingHeader.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { PrettyAddress } from 'vtex.order-details'
+import TranslateEstimate from 'vtex.shipping-estimate-translator/TranslateEstimate'
+
+const ShippingHeader = ({ shippingData, index, numPackages }) => {
+  const multipleDeliveries = (numPackages > 1)
+  return (
+    <div className="flex flex-column">
```

Comment:
`flex flex-column` is not required here since a `p` is a block element.

Reply:
Fixed.